sxiv.1

```diff
@@ -393,6 +399,18 @@ Color of the bar background. Defaults to window.foreground
 .B bar.foreground
 Color of the bar foreground. Defaults to window.background
 .TP
+.B title.prefix
+Any string literal to be used as the window title prefix. Defaults to "sxiv"
+.TP
+.B title.suffixmode
+The format of the window title suffixmode.
+.EX
+    Value  Format
+    0      Basename of file
+    1      Full path to file
+    2      Empty string
+.EE
```

Comment:
It's more natural to have 0 as empty, 1 as basename and 2 as full path.

Reply:
I think this was chosen such that 0 will be the default. I am okay with both arrangements.

Reply:
I think we should have default be empty, even if we want basename to be default, 0 should still correspond to empty. It's just much more natural.

- - -

Also let's have more readable enum names, `EMPTY`, `BASENAME`, `FULLPATH` would be nice.